subprojects/docs/src/docs/userguide/kotlin_dsl.adoc

```diff
@@ -492,11 +492,78 @@ The example above centrally defines plugin versions in the settings script's plu
 Note that alternatively you could also move the version declarations in the root project `plugins {}` block.
 
 
-// [[sec:collections]]
-// == TODO Working with collections
-// overview & api
-// === Via Kotlin delegated properties
-// === Container scope string invoke extension
+[[sec:containers]]
+== Working with containers
+
+The Gradle build model makes heavy use of containers.
+For example, Gradle core contributes `configurations` which is a container of `Configuration` objects and `tasks` which is a container of `Task` objects ; core plugins such as `java` or `distribution` contribute containers such as `sourceSets` or `distributions`.
```

Comment:
:x: spurious space before semicolon

Reply:
:o: I think should be `and` instead of `or` in the ending sentence: _core plugins such as `java` **and** `distribution` contribute containers such as `sourceSets` **and** `distributions`_